packages/cli/src/Logger.ts

```diff
@@ -112,21 +109,17 @@ export class Logger implements ILogger {
 		this.log('error', message, meta);
 	}
 
-	verbose(message: string, meta: object = {}): void {
-		this.log('verbose', message, meta);
-	}
-
 	warn(message: string, meta: object = {}): void {
 		this.log('warn', message, meta);
 	}
-}
-
-let activeLoggerInstance: Logger | undefined;
 
-export function getLogger() {
-	if (activeLoggerInstance === undefined) {
-		activeLoggerInstance = new Logger();
+	/** @deprecated: use `debug` directly from `n8n-workflow */
```

Comment:
1. Why do we need to use `debug` directly instead of through `Logger`?
2. Not seeing debug logs with `export N8N_LOG_LEVEL=debug`? [Before](https://share.cleanshot.com/Dvlv5qHH) and [after](https://share.cleanshot.com/T42J9QPB). Is this expected?

Reply:
reverted this.